### ISSUE TITLE
Beta not yet working?

### ISSUE THREAD

**Issue by xeronimus** [bricelam/ImageResizer#37]
If v3.1.0-beta1 is not supposed to work yet, you can close the issue right away :)

I see the new advanced properties, however actually resizing fails with the following error dialog:

![image](https://cloud.githubusercontent.com/assets/1777143/23413666/9f991ab0-fdd9-11e6-9c38-5a1577bbe560.png)

This is reproducable for me with different jpeg files as well as different settings ( target size, filename parameters, etc.)

The integer in the path containing *Windows\system32* changes each time, the second path *C:\p* is always the same.

My Environment:
* ImageResizer v3.1.0-beta1 (zip download from github releases)
* Windows 8.1 Enterprise 64bit

**Comment by JNavas2** [comment]
Image Resizer v3.1.0-beta1 works fine here on Windows 8.1 Pro x64.
What _exactly_ are you trying to do?
What else is installed? It looks like something is corrupting parameters.
Have you tried removing and reinstalling?

**Comment by xeronimus** [comment]
That's right, I had the stable version installed previously..

Uninstall and reinstall did not fix my problem.

can I manually remove the old shell extension?

**Comment by JNavas2** [comment]
Suggest you:
1. Uninstall beta.
2. Install stable.
3. Uninstall stable.
4. Install beta.

**Comment by scrobbleme** [comment]
Same here. The suggestion from @JNavas2  didn't worked for me. Still the issue.

**Comment by Kbird666** [comment]
This issue was happening to me too  as I had not rebooted after uninstalling Version 3 and then installing the Beta. 

I installed the Beta as the Resizer is not calculating the resize here correctly , it was in Ver.3 and is not with the Beta either , and the new Fill and Fit options don't help with the true size. 

For eg. my monitor is 2560x1440  if I do a Custom Resize to that , I get 1080x1440
with the Beta if I set Fill or stretch instead of Fit I get 1440x1440 

Like most I am on Windows 10 these days Version 1607.
